src/GraphQL/Execution/Nodes/ExecutionNode.cs

```diff
@@ -178,7 +190,7 @@ private IEnumerable<object> GeneratePath(bool preferAlias)
             {
                 pathList[--index] = node.IndexInParentNode.HasValue
                     ? GetObjectIndex(node.IndexInParentNode.Value)
-                    : preferAlias ? node.Name! : node.Field!.Name!;
+                    : node.GetNameOrAlias(preferAlias);
```

Comment:
```suggestion
                    : preferAlias ? node.Name! : (string)node.Field!.Name!;
```

Reply:
Changed to `: preferAlias ? node.Name! : node.FieldDefinition.Name;`